src/EditorFeatures/VisualBasicTest/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.vb

```diff
@@ -219,7 +219,7 @@ End structure",
 Imports System
 Imports System.Collections.Generic
 
-Structure Program
+structure Program
```

Comment:
I unified code between VB and C#.  The older VB code happened to annotate more of the file with the annotations to fix things up.  The C# code does not.  I don't think the VB approach was desirable.